content/docs/workflow-examples/escalation/escalation-eda.md

```diff
@@ -3,4 +3,4 @@ title: Event Driven Architecture Example
 date: "2024-02-28"
 ---
 
-{{< remoteMD "https://github.com/parodos-dev/serverless-workflow-examples/blob/main/escalation-eda/README.md?raw=true" >}}
```

Comment:
Were the content identical?
Anyway, nothing should point to `serverless-workflow-examples` in this repo

Reply:
>  Anyway, nothing should point to serverless-workflow-examples in this repo
why not? it's the only place where we have escalation wf

Reply:
`serverless-workflow-examples` is our sandbox, playground while this repo is for the publicly available, prod ready items

Reply:
> why not? it's the only place where we have escalation wf

I actually modified the escalation from the examples repository to the serverless-workflows repository in this PR for the escalation-EDA one.